src/transactions/test/SetTrustLineFlagsTests.cpp

```diff
@@ -112,11 +130,18 @@ TEST_CASE("set trustline flags", "[tx][settrustlineflags]")
         {
             SECTION("authorized to maintain liabilities -> authorized")
             {
-                gateway.allowMaintainLiabilities(
-                    idr, a1, TrustFlagOp::SET_TRUST_LINE_FLAGS);
-                gateway.setOptions(clearFlags(AUTH_REVOCABLE_FLAG));
+                auto market = TestMarket{*app};
```

Comment:
This test is kind of weird. Unless I am missing something, it's really testing that you can go from authorized -> authorized to maintain liabilities -> authorized, even though the title only suggests it does the second step. I think you should be able to restructure this to use `setFlagAndCheckOffer`.

Reply:
Changed the name and used two calls to `setFlagAndCheckOffer`.